core/src/main/java/org/apache/accumulo/core/iterators/FirstEntryInRowIterator.java

```diff
@@ -147,7 +147,7 @@ public IteratorOptions describeOptions() {
   @Override
   public boolean validateOptions(Map<String,String> options) {
     String o = options.get(NUM_SCANS_STRING_NAME);
-    if (o != null && !NumberUtils.isNumber(o))
+    if (o != null && !NumberUtils.isCreatable(o))
```

Comment:
I think `isParsable` makes more sense here because we need to call `Integer.parseInt` later. This isn't a perfect check, since it doesn't explicitly check that it's parse-able by the `Integer` class (it could be parse-able by `Long`, `Double`, etc.), but it's closer to what we want to check for than `isCreatable` since hex, etc. notations are definitely not appropriate for this option.